doc/cli.markdown

```diff
@@ -672,6 +678,84 @@ Examples:
 
 device
 
+# Tags
+
+## tags
+
+Use this command to list all tags for
+a particular application, device or release.
+
+This command lists all application/device/release tags.
+
+Example:
+
+	$ balena tags --application MyApp
+	$ balena tags --device 7cf02a6
+	$ balena tags --release 1234
+
+### Options
+
+#### --application, -a, --app &#60;application&#62;
+
+application name
+
+#### --device, -d &#60;device&#62;
+
+device uuid
+
+#### --release, -r &#60;release&#62;
+
+release id
+
+## tag set &#60;tagKey&#62; [value]
+
+Use this command to set a tag to an application, device or release.
+
+Examples:
+
+	$ balena tag set beta --application MyApp
+	$ balena tag set status tested --application MyApp
```

Comment:
@thgreasi is this "key" and "value", from the docs its unclear if this is setting key value pairs or multiple keys. We should make it more clear that this is to set the key value pair.

Reply:
I guess that all my brain needed to see was `tag set <tagKey> [value]`, but makes sense @shaunmulligan :+1: . Pushing a commit in a moment.